oaknational/Oak-Web-Application
# LESQ-105 LESQ-93 feat(unit and programme): update query to display unitvariant data on…

… unit list

## Description

- adds examboard to programme listing
- adds unit variance to unit listing including:
 1) display correct units for each programme
2) display correct lesson count for each unit
3) display correct unit count for each programme
4) units are displayed in all programmes if only null variant exists


## Issue(s)

Fixes #

## How to test

1. Go to /beta/teachers-2023/
2. Click on various unit listing pages
3. You should see units with unitvariance displayed

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
